integration_tests/src/main/python/delta_lake_delete_test.py

```diff
@@ -78,7 +79,7 @@ def test_delta_delete_disabled_fallback(spark_tmp_path, disable_conf):
     def setup_tables(spark):
         setup_delta_dest_tables(spark, data_path,
                                 dest_table_func=lambda spark: unary_op_df(spark, int_gen),
-                                use_cdf=False)
+                                use_cdf=False, enable_deletion_vectors=False)
```

Comment:
Why is this hard coded to false? Does it not work with them enabled? If so then we need a follow on issue to fix that.